setup.py

```diff
@@ -29,7 +29,7 @@ def readme():
 	  author = 'Nikhil Kumar Singh',
 	  author_email = 'nikhilksingh97@gmail.com',
 	  license = 'MIT',
-	  packages = ['clix'],
+	  packages = ['clix',''],
```

Comment:
Well I don't know if this is the best way to do this. Can you send me the link where you found this solution?